modules/serverless-configuring-runtimeclass-name.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * serverless/knative-serving/config-applications/deployment-resources.adoc
```

Comment:
This is the actual path now that it's a stand-alone branch, yeah? I.e., no `serverless/` folder:
```suggestion
// * knative-serving/config-applications/deployment-resources.adoc
```